test/infra/configs/config-management-operator.yaml

```diff
@@ -0,0 +1,279 @@
+# ----- configmanagement_v1_configmanagement.yaml -----
```

Comment:
Could you add a comment at the top explaining where this came from?

Reply:
Done